02.calendar/cal.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'date'
+
+options = ARGV.getopts('y:', 'm:')
+
+input_year = options["y"]
+input_month = options["m"]
+
+# -yが入力されなかった場合は今年を指定
+target_year = input_year.nil? ? Date.today.year : input_year.to_i
+# -mが入力されなかった場合は今月を指定
+target_month = input_month.nil? ? Date.today.month : input_month.to_i
+
+# 対象月の初日
+first_date_of_month = Date.new(target_year, target_month, 1)
```

Comment:
```suggestion
first_date = Date.new(target_year, target_month, 1)
```

修飾語はなくても大丈夫かなと思います
https://bootcamp.fjord.jp/pages/avoid-too-specific-var-name